test/IRGen/tsan_coroutines.swift

```diff
@@ -23,7 +22,7 @@ extension Foobar {
     }
 
     // We used to crash emitting the subscript function.
-    // CHECK: define swiftcc { i8*, %T15tsan_coroutines1CC* } @"$s15tsan_coroutines6FoobarVyAA1CCAC5IndexVcir"
+    // CHECK: define{{( protected)?}} swiftcc { i8*, %T15tsan_coroutines1CC* } @"$s15tsan_coroutines6FoobarVyAA1CCAC5IndexVcir"
```

Comment:
This needs a check for `dllexport` on Windows.

Reply:
Fixed.